### PR TITLE
fixed progress percentage floating issue

### PR DESCRIPTION
**Pull request by saurTarento**
Plz review and let me know if any changes are required.

### CODE REVIEW

File: project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.scss [hunk 1]
```diff
@@ -20,9 +22,11 @@
   position: absolute;
   right: 5%;
 }
+
 .image {
   height: 20px !important;
```
Comment (revathip) [inline project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.scss:27]: Please avoid !important

File: project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.scss [hunk 2]
```diff
@@ -175,6 +179,11 @@ span.content-top {
   margin-top: 4px;
   margin-left: 6px;
 }
+
 .w-100 {
   width: 100%;
 }
+
+span.resource-progress {
+  position: static !important;
```
Comment (revathip) [inline project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.scss:188]: try without !important

File: project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.scss (hunk 1 above)
Comment (revathip) [reply project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.scss:27]: Fix it

Comment (saurTarento) [reply project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.scss]: Fixed. Plz check

File: project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.scss (hunk 2 above)
Comment (saurTarento) [reply project/ws/viewer/src/lib/components/viewer-toc/viewer-toc.component.scss]: Removed.. plz check

**Review by revathip — APPROVED** [review]

**Pull request merged**